Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -983,6 +1150,46 @@ def top_tags_results_command():
     })
 
 
+def search_ip_command(ip):
+    indicator_type = 'IP'
+    ip_list = argToList(ip)
+    for ip_address in ip_list:
```

Comment:
the loop should be only for the `search_indicator` function, the rest should handle a list of results